shopfloor_mobile/static/wms/src/components/packaging-qty-picker.js

```diff
@@ -280,9 +280,16 @@ export var PackagingQtyPicker = Vue.component("packaging-qty-picker", {
                     <v-col cols="2" md="2" v-if="!readonly">
                         <span class="qty-todo">/ {{ qty_todo_by_pkg[pkg.id] }}</span>
                     </v-col>
-                    <v-col>
-                        <div class="pkg-name"> {{ pkg[pkgNameKey] }}</div>
-                        <div v-if="contained_packaging[pkg.id]" class="pkg-qty">(x{{ contained_packaging[pkg.id].qty }} {{ contained_packaging[pkg.id].pkg.name }})</div>
+                    <v-col cols="4">
+                        <div :class="qty_by_pkg[pkg.id] > 0 ? 'pkg-name font-weight-bold' : 'pkg-name'"> {{ pkg[pkgNameKey] }}</div>
+                        <div v-if="contained_packaging[pkg.id]" :class="qty_by_pkg[pkg.id] > 0 ? 'pkg-qty font-weight-bold' : 'pkg-qty'">(x{{ contained_packaging[pkg.id].qty }} {{ contained_packaging[pkg.id].pkg.name }})</div>
+                    </v-col>
+                    <v-col cols="2" v-if="_.result(contained_packaging[pkg.id], 'pkg.icon.url', '')">
```

Comment:
IMO the icon should stay before the input... but that's probably only a matter of taste

Reply:
You should append `/30x30` to the URL to get a resized image.

Reply:
I did but it doesn't change anything in the frontend, the image adjusts to the size of the row. This is why I'm setting its size in a CSS class.

Regarding the icon, I put it to the right so that it's closer to the name (I think it's more readable like this), but it's only a preference too.